.cascade-code/Chapman.edu/_components/left_nav/left_nav_no_siblings.vtl

```diff
@@ -0,0 +1,171 @@
+## Formerly LeftNavNoSiblings. Primarily used for index pages.
+## New left nav format that works without a block
+
+#set ( $parentDir = $currentPage.parentFolder )
+
+## Builds the left navigation for non-index pages (only shows the title)
+#macro ( buildNonIndexLeftNav )
+    #buildNonIndexLeftNavTitle()
+    ## Needs to have empty <ul> tags because otherwise title does not have a bottom margin
+    ## Without it, other left col widgets will show up directly under the left nav with no space
+    <ul></ul>
```

Comment:
does this make sense? it's kind of an ugly weird thing that has to be here

Reply:
Ok with me. Especially with commment.

Reply:
I checked if empty UL tags cause any accessibility issues and they don't